policyengine_us/parameters/gov/states/az/hhs/tanf/eligibility/income/earned/flat.yaml

```
@@ -0,0 +1,11 @@
+description: Arizona disregards this amount of earned income under the cash assistance program.
+metadata:
+  unit: currency-USD
+  label: Arizona Cash Assistance flat earned income disregard
+  period: year
```

right? same for percentage
```suggestion
  period: month
```